src/modules/auth/auth.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Body, Controller, HttpStatus, Post, Request, Res } from '@nestjs/common';
+import { Response } from 'express';
+import { CreateUserDTO } from './dto/create-user.dto';
+import { skipAuth } from '../../helpers/skipAuth';
+import AuthenticationService from './auth.service';
+
+@Controller('auth')
+export default class RegistrationController {
+  constructor(
+    private authService: AuthenticationService,
+  ) { }
+
+  @skipAuth()
```

Comment:
Yes it is, that is because the authguard was enabled globally as protected routes are more than public ones, so this is a decorator to make a route public